test/controllers/organization_memberships_controller_test.exs

```diff
@@ -25,96 +25,70 @@ defmodule CodeCorps.OrganizationMembershipControllerTest do
 
   describe "index" do
     test "lists all resources", %{conn: conn} do
-      membership_1 = insert(:organization_membership)
-      membership_2 = insert(:organization_membership)
+      [membership_1, membership_2] = insert_pair(:organization_membership)
 
       path = conn |> organization_membership_path(:index)
+      response = conn |> get(path) |> json_response(200)
 
-      data = conn |> get(path) |> json_response(200) |> Map.get("data")
-      assert data |> length == 2
-
-      [first_result, second_result] = data
-      assert first_result["id"] == "#{membership_1.id}"
-      assert second_result["id"] == "#{membership_2.id}"
+      assert ids_from_response(response) == [membership_1.id, membership_2.id]
     end
 
     test "lists all resources for specified organization", %{conn: conn} do
       organization = insert(:organization)
-      membership_1 = insert(:organization_membership, organization: organization)
-      membership_2 = insert(:organization_membership, organization: organization)
+      [membership_1, membership_2] = insert_pair(:organization_membership, organization: organization)
       insert(:organization_membership)
 
       path = conn |> organization_organization_membership_path(:index, organization)
+      response = conn |> get(path) |> json_response(200)
 
-      data = conn |> get(path) |> json_response(200) |> Map.get("data")
-      assert data |> length == 2
-
-      [first_result, second_result] = data
-      assert first_result["id"] == "#{membership_1.id}"
-      assert second_result["id"] == "#{membership_2.id}"
+      assert ids_from_response(response) == [membership_1.id, membership_2.id]
```

Comment:
I think `ids_from_response` is a great addition! It makes testing easier to write *and* read 🐎